homeassistant/components/mysensors/sensor.py

```diff
@@ -46,63 +47,107 @@
 from .const import MYSENSORS_DISCOVERY, DiscoveryInfo
 from .helpers import on_unload
 
-SENSORS: dict[str, list[str | None] | dict[str, list[str | None]]] = {
-    "V_TEMP": [None, None, DEVICE_CLASS_TEMPERATURE, STATE_CLASS_MEASUREMENT],
-    "V_HUM": [
-        PERCENTAGE,
-        "mdi:water-percent",
-        DEVICE_CLASS_HUMIDITY,
-        STATE_CLASS_MEASUREMENT,
-    ],
-    "V_DIMMER": [PERCENTAGE, "mdi:percent", None, None],
-    "V_PERCENTAGE": [PERCENTAGE, "mdi:percent", None, None],
-    "V_PRESSURE": [None, "mdi:gauge", None, None],
-    "V_FORECAST": [None, "mdi:weather-partly-cloudy", None, None],
-    "V_RAIN": [None, "mdi:weather-rainy", None, None],
-    "V_RAINRATE": [None, "mdi:weather-rainy", None, None],
-    "V_WIND": [None, "mdi:weather-windy", None, None],
-    "V_GUST": [None, "mdi:weather-windy", None, None],
-    "V_DIRECTION": [DEGREE, "mdi:compass", None, None],
-    "V_WEIGHT": [MASS_KILOGRAMS, "mdi:weight-kilogram", None, None],
-    "V_DISTANCE": [LENGTH_METERS, "mdi:ruler", None, None],
-    "V_IMPEDANCE": ["ohm", None, None, None],
-    "V_WATT": [POWER_WATT, None, DEVICE_CLASS_POWER, STATE_CLASS_MEASUREMENT],
-    "V_KWH": [
-        ENERGY_KILO_WATT_HOUR,
-        None,
-        DEVICE_CLASS_ENERGY,
-        STATE_CLASS_MEASUREMENT,
-    ],
-    "V_LIGHT_LEVEL": [PERCENTAGE, "mdi:white-balance-sunny", None, None],
-    "V_FLOW": [LENGTH_METERS, "mdi:gauge", None, None],
-    "V_VOLUME": [VOLUME_CUBIC_METERS, None, None, None],
-    "V_LEVEL": {
-        "S_SOUND": [SOUND_PRESSURE_DB, "mdi:volume-high", None, None],
-        "S_VIBRATION": [FREQUENCY_HERTZ, None, None, None],
-        "S_LIGHT_LEVEL": [
-            LIGHT_LUX,
-            "mdi:white-balance-sunny",
-            DEVICE_CLASS_ILLUMINANCE,
-            STATE_CLASS_MEASUREMENT,
-        ],
-    },
-    "V_VOLTAGE": [
-        ELECTRIC_POTENTIAL_VOLT,
-        "mdi:flash",
-        DEVICE_CLASS_VOLTAGE,
-        STATE_CLASS_MEASUREMENT,
-    ],
-    "V_CURRENT": [
-        ELECTRIC_CURRENT_AMPERE,
-        "mdi:flash-auto",
-        DEVICE_CLASS_CURRENT,
-        STATE_CLASS_MEASUREMENT,
-    ],
-    "V_PH": ["pH", None, None, None],
-    "V_ORP": [ELECTRIC_POTENTIAL_MILLIVOLT, None, None, None],
-    "V_EC": [CONDUCTIVITY, None, None, None],
-    "V_VAR": ["var", None, None, None],
-    "V_VA": [POWER_VOLT_AMPERE, None, None, None],
+SENSORS: dict[str, SensorEntityDescription] = {
```

Comment:
This is only used for the lookup in `_get_entity_description`
Since the dict key and the `description.key` is identical for all  of them it will be cleaner to just use a tuple here instead of a dict and will how most other integrations that have converted to `SensorEntityDescription` does it